Holovibes/sources/compute/rendering.cc

```diff
@@ -227,10 +236,28 @@ void Rendering::insert_apply_contrast(WindowKind view)
         });
 }
 
-void Rendering::insert_compute_autocontrast(std::atomic<bool>& autocontrast_request,
-                                            std::atomic<bool>& autocontrast_slice_xz_request,
-                                            std::atomic<bool>& autocontrast_slice_yz_request,
-                                            std::atomic<bool>& autocontrast_filter2d_request)
+/*! \brief Tell if the contrast should be applied
+ *
+ * \param request The request for autocontrast
+ * \param queue The accumulation queue
+ * \return true if the contrast should be applied
+ */
+inline bool apply_contrast(bool request, const std::unique_ptr<Queue>& queue)
```

Comment:
Maybe the name is a bit misleading; for me the function should have a name that reflects its 'testing' purpose
Like 'check_autocontrast' or 'do_apply_autocontrast'